helpers/helpers.go

```diff
@@ -4,16 +4,23 @@ import (
 	"crypto/sha256"
 	"encoding/hex"
 	"encoding/json"
+	"fmt"
 	"net/http"
 	"os"
 
+	"github.com/ndv6/tsaving/constants"
+
 	"github.com/ndv6/tsaving/models"
 )
 
 //untuk ngehandle error"
 func HTTPError(w http.ResponseWriter, status int, errorMessage string) {
 	w.WriteHeader(status)
-	json.NewEncoder(w).Encode(map[string]string{"error": errorMessage})
+	w, resp, err := NewResponseBuilder(w, false, errorMessage, make(map[string]string))
+	if err != nil {
+		json.NewEncoder(w).Encode(map[string]string{"error": constants.CannotEncodeResponse})
+	}
+	fmt.Fprintln(w, resp)
```

Comment:
i think need to set content type to json write the response

Reply:
The solution for now is to add Header manually at the beginning of handlerfunc for every handler